larpandoracontent/LArCheating/CheatingNeutrinoIdTool.cc

```diff
@@ -19,8 +19,10 @@ using namespace pandora;
 namespace lar_content
 {
 
-void CheatingNeutrinoIdTool::SelectOutputPfos(const SliceHypotheses &nuSliceHypotheses, const SliceHypotheses &crSliceHypotheses, PfoList &selectedPfos)
+void CheatingNeutrinoIdTool::SelectOutputPfos(const MasterAlgorithm *const pAlgorithm, const SliceHypotheses &nuSliceHypotheses, const SliceHypotheses &crSliceHypotheses, PfoList &selectedPfos)
```

Comment:
Presumably no need for this to be a MasterAlgorithm instance, rather than an Algorithm instance? Maybe ultimately don't need to take an Algorithm at all, but probably OK for first iteration. (And similar, below.)